### PR TITLE
[CI] Added license checker

### LINKED ISSUE #55: Enforce copyright header
Using `github.com/u-root/u-root/tools/checklicenses` with config

```
{
    "gopkg": "github.com/coredhcp/coredhcp",
    "licenses": [
        [
            "^// Copyright 2015-2018 the CoreDHCP Authors\\. All rights reserved",
            "// This source code is licensed under the MIT license found in the",
            "// LICENSE file in the root directory of this source tree\\."
        ]
    ],
    "accept": [
        ".*\\.go"
    ],
    "reject": []
}
```

### PR DESCRIPTION
**Pull request by insomniacslk**
Fixes #55 

Signed-off-by: Andrea Barberio <insomniac@slackware.it>

### CODE REVIEW

**Comment by insomniacslk** [comment]
The CI will fail because there are various files without license header. Once fixed I'll merge this

**Comment by Natolumin** [comment]
I think you forgot the `.travis/config.json` file. maybe name it a more explicit `license_header.json` or something since this isn't really a config for travis in general

**Comment by insomniacslk** [comment]
whoops. Done!

File: .travis/checklicenses_config.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "gopkg": "github.com/coredhcp/coredhcp",
+    "licenses": [
+        [
+            "^// Copyright 2015-2018 the CoreDHCP Authors\\. All rights reserved",
```
Comment (Natolumin) [inline .travis/checklicenses_config.json]: nit: 2019

Comment (insomniacslk) [reply .travis/checklicenses_config.json]: bad copy-pasta, it should be 2018-present

**Comment by insomniacslk** [comment]
> will you add the headers to the other files in the same PR or a separate one ?

separate PRs, and I'll rebase until the CI is green

File: .travis/checklicenses_config.json (hunk 1 above)
Comment (insomniacslk) [reply .travis/checklicenses_config.json]: fixed

Comment (insomniacslk) [reply .travis/checklicenses_config.json]: fixed

**Comment by insomniacslk** [comment]
Headers added in https://github.com/coredhcp/coredhcp/pull/57

**Comment by insomniacslk** [comment]
rebased on top of #57

**Pull request merged**